docker-compose.yaml

```diff
@@ -8,9 +8,9 @@ services:
     environment:
       - DEFI_ADAPTERS_PROVIDER_ETHEREUM
       - DEFI_ADAPTERS_PROVIDER_OPTIMISM
-      - DEFI_ADAPTERS_PROVIDER_BSC=
+      - DEFI_ADAPTERS_PROVIDER_BSC
       - DEFI_ADAPTERS_PROVIDER_POLYGON
-      - DEFI_ADAPTERS_PROVIDER_FANTOM=
+      - DEFI_ADAPTERS_PROVIDER_FANTOM
```

Comment:
No more cheating, local env needs to be as close as the deployed ones as possible.